core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -118,6 +118,7 @@
 
   {# TODO(bhenning): Remove these once RouterServices is decoupled from the exploration editor #}
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationPropertyService.js"></script>
```

Comment:
Is this required? (I'm not sure, do check it)

Reply:
@DubeySandeep   RouterService is being imported as well which depends on ExplorationStatesService and ExplorationInitStateNameService so I think it should be .
Please help me on this.

Reply:
Great! that's what I wanted you to check! :)
Also, can you check do we still need `<script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>` ?

Reply:
I don't think we do as it is not required by the other files imported.
I made the change and ran the presubmit checks,it passed.

Reply:
Thanks for checking! :)